src/Session/Access/Donation.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Give\Session\Access;
+
+use DateTime;
+use Give\ValueObjects\Session\Donation as DonationObject;
+
+/**
+ * Class Donation
+ *
+ * @package Give\Session\Access
+ *
+ * @property-read string         $sessionKey
+ * @property-read DonationObject $dataObj
+ * @property-read array          $renameTo
```

Comment:
You don't need to add these if the properties are fixed. These docblocks are only necessary for dynamic properties and methods.

Reply:
fixed: https://github.com/impress-org/givewp/compare/1e4adddcb256...2a7b11961de2